packages/terra-functional-testing/src/services/wdio-visual-regression-service/methods/SaveScreenshot.js

```diff
@@ -1,8 +1,8 @@
 import fs from 'fs-extra';
-import logger from '@wdio/logger';
+import Logger from '@cerner/terra-cli/lib/utils/Logger';
```

Comment:
Would it be possible to export the logger from index.js?
```suggestion
import { Logger } from '@cerner/terra-cli';
```

Reply:
Good call: https://github.com/cerner/terra-toolkit/pull/486/commits/0be03600372978253e258db868a15297812337e0